source/clear-linux/concepts/autospec-about.rst

```diff
@@ -1,112 +1,106 @@
-.. _autospec-about: 
+.. _autospec-about:
 
 Autospec
 ########
 
-.. _incl-autospec-overview:
+A standard RPM build process using ``rpmbuild`` requires a tarball and .spec
+file to start. The .spec provides information about a package, how to
+build it, as well as what should be installed and where.
 
```

Comment:
Please reverse the order of these two paragraphs. Lines 10-12 are para. #1; lines 6-8 are para. #2.